render/cursor.go

```diff
@@ -26,11 +26,13 @@ func deltaMarkup(fromCur, toCur ecma48.Cursor) string {
 	to := toCur.Style
 	from := fromCur.Style
 
-	if to.Bg.ColorMode != from.Bg.ColorMode || to.Bg.Code != from.Bg.Code {
+	differentBgCode := (to.Bg.ColorMode != ecma48.ColorNone || true) && to.Bg.Code != from.Bg.Code
```

Comment:
`(to.Bg.ColorMode != ecma48.ColorNone || true)` looks like it always returns true no matter what. So I think it's just `to.Bg.Code != from.Bg.Code`